lib/did_you_mean/test_helper.rb

```diff
@@ -1,7 +1,7 @@
 module DidYouMean
   module TestHelper
     def assert_suggestion(array, expected)
-      assert_equal [expected], array, "Expected #{array.inspect} to only include #{expected.inspect}"
+      assert_equal Array(expected), array, "Expected #{array.inspect} to only include #{expected.inspect}"
```

Comment:
If you want to assert multiple suggestions, consider adding `assert_suggestions` method instead of changing simgular `assert_suggestion`. Also, don't include multiple changes in one pull request.